trainings/template.md

```diff
@@ -0,0 +1,30 @@
+# Training title (e.g. Sourcegraph 101)
+
+**Topic:** This training is focused on...
+
+## Unit 1 title (e.g. Getting familiar with Sourcegraph)
+
+**Learning goals:** After completing this unit, you'll be able to... (A training module may have multiple units. Each one will have its own goals, sections, conclusion, resources, and evaluation.)
+
+### Section 1 title (e.g. What is Sourcegraph?)
+
+Section 1 content. All sections together should be between 500 and 1500 words per unit for customer-facing written content. Units may have multiple sections.
+
```

Comment:
Yeah, seems fine! I think with webinar trainings folks typically don't do that in flow (just since people don't always have access yet) but I think it's good to shoot for at the very least!